web/packages/teleterm/src/ui/uri.ts

```diff
@@ -25,34 +25,22 @@ import type { RouteProps } from 'react-router';
  * These are for identifying a specific resource within a root cluster.
  */
 
-type RootClusterId = string;
-type LeafClusterId = string;
-type ServerId = string;
-type KubeId = string;
-type DbId = string;
-type AppId = string;
-export type RootClusterUri = `/clusters/${RootClusterId}`;
-export type RootClusterServerUri =
-  `/clusters/${RootClusterId}/servers/${ServerId}`;
-export type RootClusterKubeUri = `/clusters/${RootClusterId}/kubes/${KubeId}`;
-export type RootClusterDatabaseUri = `/clusters/${RootClusterId}/dbs/${DbId}`;
-export type RootClusterAppUri = `/clusters/${RootClusterId}/apps/${AppId}`;
+export type RootClusterUri = string;
```

Comment:
Could you leave a TODO comment here? Otherwise it's going to be quite confusing if someone looks at these types with no context. ;)

I assume we're leaving those in because 1) ripping them out is too much work 2) they still carry some useful information 3) we might be able to add them back in the future; is that correct?

Reply:
I left the types mostly because they are still useful, `clusterUri: RootClusterUri` is still better than `uri: string`.
A TODO added.